### PR TITLE
added support for postgresql on linux

### PR DESCRIPTION
**Pull request by rishwanth1995**

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -362,6 +421,8 @@ Building:
    afflib support:                        $ax_afflib
    libewf support:                        $ax_libewf
    zlib support:                          $ax_zlib
+   libvhdi support:                       $ax_libvhdi
+   libvmdk support:                       $ax_libvmdk
```
Comment (bcarrier) [inline configure.ac]: Add postgresql

File: tsk/auto/db_postgresql.cpp [hunk 2]
```diff
@@ -125,11 +118,16 @@ PGconn* TskDbPostgreSQL::connectToDatabase(TSK_TCHAR *dbName) {
     PQescapeString(&userName_sql[0], userName, strlen(userName));
     PQescapeString(&password_sql[0], password, strlen(password));
     PQescapeString(&hostName_sql[0], hostNameOrIpAddr, strlen(hostNameOrIpAddr));
+#ifdef TSK_WIN32
+	snprintf(connectionString, 1024, "user=%s password=%s dbname=%S host=%s port=%s", userName_sql, password_sql, dbName, hostName_sql, hostPort);
```
Comment (bcarrier) [inline tsk/auto/db_postgresql.cpp]: Can't we use the %PRI.... for this?

File: tsk/auto/db_postgresql.cpp [hunk 3]
```diff
@@ -148,25 +146,28 @@ TSK_RETVAL_ENUM TskDbPostgreSQL::createDatabase(){
...
-    // If you use single quotes, PostgreSQL will convert db name to lower case. If database was created using double quotes 
+    // If you use single quotes, PostgreSQL will convert db name to lower case. If database was created using double quotes
     // you now need to always use double quotes when referring to it (e.dg when deleting database).
     char createDbString[1024];
+#ifdef TSK_WIN32
     snprintf(createDbString, 1024, "CREATE DATABASE \"%S\" WITH ENCODING='UTF8';", m_dBName);
-
-    PGresult *res = PQexec(serverConn, createDbString); 
+#else
+	snprintf(createDbString, 1024, "CREATE DATABASE \"%s\" WITH ENCODING='UTF8';", m_dBName);
```
Comment (bcarrier) [inline tsk/auto/db_postgresql.cpp]: Same as above.  Can't we use PRIttocTSK?

File: configure.ac (hunk 1 above)
Comment (rishwanth1995) [reply configure.ac]: sure

File: tsk/auto/db_postgresql.cpp (hunk 2 above)
Comment (rishwanth1995) [reply tsk/auto/db_postgresql.cpp]: It works fine on linux and then it complains on windows. So, only I added the ifdef logic.

**Pull request merged**